src/codecs/tga/decoder.rs

```diff
@@ -153,6 +153,7 @@ impl<R: Read + Seek> TgaDecoder<R> {
             (0, 24, true) => self.color_type = ColorType::Rgb8,
             (8, 8, false) => self.color_type = ColorType::La8,
             (0, 8, false) => self.color_type = ColorType::L8,
+            (8, 0, false) => self.color_type = ColorType::L8, // alpha-only image is treated as L8
```

Comment:
This would be less confusing if the [`original_color_type`](https://docs.rs/image/0.23.12/image/trait.ImageDecoder.html#method.original_color_type) would indicate this as `A8` or a similarly named new variant of `ExtendedColorType` that clarifies that the color has been converted, or rather reinterpreted, as Luminance data.

Could you override that method in the impl of `ImageDecoder` below?

Reply:
Will do. Originally I thought that adding A8 as a native format may set some expectations that I wasn't prepare to commit to implementing (such as full saving/loading) :) If it's acceptable to just add A8, do conversion and store original format, then I'm all for it.

Reply:
Only add it to `ExtendedColorType`, not `ColorType`. That makes it clear that it is not fully and not directly supported but you can grab the information anyways for those image formats that care about the use case, including tga.